.github/workflows/node.js.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Test
+
+on:
+  push:
+    branches: [ "main", "v3" ]
+  pull_request:
+    branches: [ "main", "v3" ]
```

Comment:
the "v3" branch inclusion is temporary while we're working on the next version of the CLI

Reply:
Do we want to run this and circle on main while we're working out of v3?

Reply:
good point - let me turn it off for main